packages/waku/src/lib/plugins/vite-plugin-rsc-delegate.ts

```diff
@@ -1,25 +1,27 @@
 import path from 'node:path';
-import type { Plugin } from 'vite';
+import type { Plugin, TransformResult, ViteDevServer } from 'vite';
 import * as swc from '@swc/core';
 
-import { encodeFilePathToAbsolute } from '../utils/path.js';
-
 // import { CSS_LANGS_RE } from "vite/dist/node/constants.js";
 const CSS_LANGS_RE =
   /\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)(?:$|\?)/;
 
 export function rscDelegatePlugin(
-  importCallback: (source: string) => void,
+  importCallback: (source: string | TransformResult) => void,
 ): Plugin {
   let mode = 'development';
   let base = '/';
+  let server: ViteDevServer;
   return {
     name: 'rsc-delegate-plugin',
     configResolved(config) {
       mode = config.mode;
       base = config.base;
     },
-    transform(code, id) {
+    configureServer(_server) {
+      server = _server;
```

Comment:
We use `_` prefix specifically for "unused" values.

```suggestion
    configureServer(s) {
      server = s;
```

Reply:
Or
```suggestion
    configureServer(instance) {
      server = instance;
```

```suggestion
    configureServer(serverInstance) {
      server = serverInstance;
```